apps/meteor/tests/end-to-end/api/05-chat.js

```diff
@@ -1101,6 +1101,237 @@ describe('[Chat]', function () {
 				})
 				.end(done);
 		});
+
+		describe('customFields', () => {
```

Comment:
Can we add a test for the `chat.update` endpoint too?

Reply:
They are there already